documentation/builders/update.md

```diff
@@ -12,6 +12,98 @@ Restore your old files after the new installation was successful and check if ne
 $ diff shared/settings/jukebox.yaml resources/default-settings/jukebox.default.yaml
 ```
 
+## Manually upgrade to the latest version
+
+> [!CAUTION]
+> For optimal system updates, it is strongly recommended to utilize the upgrade feature when transitioning to the next version. Manual updates may necessitate specific migration steps and, if overlooked, could result in system failure.
+
+If you only want to update a few recent commits, this following explanation outlines the steps to do so
+
+Typically, 4 steps need to be considered
+
+1. Backup Local Changes (Optional)
+1. Pull the latest version from Github
+1. Replace the Web App with the most recent build
+1. Optional: Update the config files
+
+### Fetch the most recent version from Github
+
+First, SSH into your Phoniebox.
+
+```bash
+cd ~/RPi-Jukebox-RFID/
+```
+
+Second, get the latest version from Github. Depending on your proficiency with Git, you can also checkout a specific branch or version.
+Be aware, in case you have made changes to the software, stash them to keep them safe.
+
+1. Backup Local Changes (Optional):
+    - Stash your local changes:
+
+        ```bash
+        git stash push -m "Backup before pull"
+        ```
+
+    - Create a Backup Branch:
+
+        ```bash
+        git branch backup-before-pull
+        ```
+
+1. Pull Latest Changes:
+
+   ```bash
+   git pull
+   ```
+
+1. Update Web App:
+    1. Backup the current webapp build
+
+    ```bash
+    cd ~/RPi-Jukebox-RFID/src/webapp
+    mv build build-backup
+    ```
+
+    1. Go to the [Github Release page](https://github.com/MiczFlor/RPi-Jukebox-RFID/releases) find the latest `Pre-release` release (typically Alpha).
```

Comment:
The "latest" might not be sufficient.
If the user is on the main branch using the develop build might break the webapp.

It should differentiated on which branch the user is currently and only use the matching artefacts.

Reply:
I'd recommend this then only for users on develop branch, as this is not an official way to "Upgrade" the Phoniebox.
I don't want to go in such a depth here to be honest and keep it simple.

Reply:
Agree, this should be handled with caution.
If we remeber to do so we could still note "breaking" changes in the commit details.

Hopefully we can resolve this with the installation process upgrade :)